api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java

```diff
@@ -11,6 +11,8 @@
  */
 package org.openmrs.module.eptsreports.reporting.library.cohorts;
 
+import static org.openmrs.module.eptsreports.reporting.utils.EptsReportConstants.*;
```

Comment:
Can this be done individual imports as oppossed to use of wild cards?

Reply:
We did this way cause the module was failing to initialize in the server using the individual import